afs/plugins/init-workflow.json

```diff
@@ -15,6 +15,15 @@
                 "circleColor": "#b0adf3",
                 "desc": "A research effort, study, or conservation project"
             },
+            {
+                "workflowid": "ef3a374c-f795-4317-aaee-83e1493eabb6",
+                "slug": "project-collection-workflow",
+                "name": "Update Project Collection",
+                "icon": "fa fa-files-o",
+                "bgColor": "#3e89cd",
+                "circleColor": "#6ea7d9",
+                "desc": "Review digital data files generated by a lab instrument"
```

Comment:
Update description to read "Add objects to an existing project"

Reply:
updated